From: James McCoy <jamessan@debian.org>
Date: Wed, 1 Aug 2018 20:44:36 -0400
Subject: apr-abi

Bug #266170: Use SOVERSION 1 for apr 1.x, because the apr 0.9.x -> 1.x
Bug #266170: Use SOVERSION 1 for apr 1.x, because the apr 0.9.x -> 1.x
ABI change leaks through the libsvn ABIs (because we use apr_off_t,
which is now 64 bits on all platforms).

Upstream refused this (http://svn.haxx.se/dev/archive-2006-03/0522.shtml
for thread), but fortunately when it came time to support apr 2.x in svn
1.7, they skipped SOVERSION 1 in consideration of Debian/Ubuntu, so we'll
be ABI-compatible again if and when we move to apr-2.x.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 31f5368..282ecc3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -106,7 +106,7 @@ if test `expr $apr_version : 2` -ne 0; then
   AC_SUBST(SVN_APRUTIL_LIBS)
   SVN_APR_MAJOR_VERSION=2
 else
-  svn_lib_ver=0
+  svn_lib_ver=1
   APU_VER_REGEXES=["1\.[3-9]\."]
   SVN_LIB_APRUTIL($APU_VER_REGEXES)
   SVN_APR_MAJOR_VERSION=1
